### PR TITLE
Touchups

### PR DESCRIPTION
**Pull request by brockboland**
These tweaks came out of the branch where I had been working on switching the paging accessory view to autolayout. I haven't finished that yet, but I'd like to get these improvements merged at the very least.

@vokal/ios-developers quick review please?

### CODE REVIEW

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 1]
```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
...
+            self.tableView.frame.size.width,
+            DefaultAccessoryHeight};
+        self.footerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:footerFrame];
     }
     
     [self.tableView addSubview:self.footerView];
     
-    [self.tableView addObserver:self forKeyPath:@"contentSize" options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld | NSKeyValueObservingOptionPrior context:NULL];
+    [self.tableView addObserver:self
+                     forKeyPath:VOKKeyForInstanceOf(UITableView, contentSize)
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: `VOKKeyForObject(self.tableView, contentSize)`?

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 2]
```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
...
+                        options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld | NSKeyValueObservingOptionPrior
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: Why are we using `NSKeyValueObservingOptionNew` and `NSKeyValueObservingOptionOld` when it doesn't look like we're using the new/old values from the change dictionary?

Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: `NSKeyValueObservingOptionPrior` causes the notification to be sent both prior to the change and after the change, but it looks like we're carrying out the same action both times.  Should we either not be using `NSKeyValueObservingOptionPrior` or checking the value in the change dictionary for the key `NSKeyValueChangeNotificationIsPriorKey`?

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 3]
```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
 {
     //Attach given views, or generate default views.
     if (!self.headerView) {
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:(CGRect){0, -30, self.tableView.frame.size.width, 30}];
+        CGRect headerFrame = (CGRect){0, -DefaultAccessoryHeight, self.tableView.frame.size.width, DefaultAccessoryHeight};
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: I'd strongly prefer either `CGRectMake()` or `(CGRect){ .origin = …, .size = … }` (using the labels on this kind of literal/initializer), here and wherever else this construct is used.

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 4]
```diff
@@ -171,8 +183,7 @@ - (void)triggerAction:(VOKPagingResultsAction)action
 
 - (void)dealloc
 {
-    NSLog(@"Page controller dealloc'd %@", self);
-    [self.tableView removeObserver:self forKeyPath:@"contentSize"];
+    [self.tableView removeObserver:self forKeyPath:VOKKeyForInstanceOf(UITableView, contentSize)];
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: `VOKKeyForObject(self.tableView, contentSize)`?

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 5]
```diff
@@ -83,10 +92,13 @@ - (void)cleanUpPageController
...
 
-- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context
+- (void)observeValueForKeyPath:(NSString *)keyPath
+                      ofObject:(id)object
+                        change:(NSDictionary *)change
+                       context:(void *)context
 {
-    if ([keyPath isEqualToString:@"contentSize"]) {
-        [self.footerView setFrame:(CGRect){0, MAX(self.tableView.contentSize.height, self.tableView.bounds.size.height), self.footerView.frame.size}];
+    if ([keyPath isEqualToString:VOKKeyForInstanceOf(UITableView, contentSize)]) {
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: `VOKKeyForObject(self.tableView, contentSize)`?

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 6]
```diff
@@ -83,10 +92,13 @@ - (void)cleanUpPageController
...
+        self.footerView.frame = (CGRect){0, MAX(self.tableView.contentSize.height, self.tableView.bounds.size.height), self.footerView.frame.size};
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: Maybe one of these?
```objc
self.footerView.frame = ({
    CGRect frame = self.footerView.frame;
    frame.origin.x = 0;
    frame.origin.y = MAX(self.tableView.contentSize.height, self.tableView.bounds.size.height);
    frame;
});
```
```objc
self.footerView.frame = (CGRect){ .origin = { 0, MAX(self.tableView.contentSize.height, self.tableView.bounds.size.height) }, .size = self.footerView.frame.size };
```

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m [hunk 7]
```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
 {
     //Attach given views, or generate default views.
     if (!self.headerView) {
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:(CGRect){0, -30, self.tableView.frame.size.width, 30}];
+        CGRect headerFrame = (CGRect){0, -DefaultAccessoryHeight, self.tableView.frame.size.width, DefaultAccessoryHeight};
+        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:headerFrame];
     }
     
-    [self.headerView setFrame:(CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size}];
+    self.headerView.frame = (CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size};
```
Comment (vokal-isaac) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: Maybe one of these?
```objc
self.headerView.frame = ({
    CGRect frame = self.headerView.frame;
    frame.origin.x = 0;
    frame.origin.y = -self.headerView.frame.size.height;
    frame;
});
```
```objc
self.headerView.frame = (CGRect){ .origin = { 0, -self.headerView.frame.size.height }, .size = self.headerView.frame.size };
```

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m (hunk 2 above)
Comment (brockboland) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: These are removed in #116

File: Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m (hunk 3 above)
Comment (brockboland) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: I didn't look to see how these got in here in the first place, but I was trying to make minimalistic changes in this one. But, I agree: I'll change them.

Comment (brockboland) [inline Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m]: Wait, no: these were mostly removed in #116 too

**Pull request merged**